src/extension/module.js

```diff
@@ -1864,6 +1970,31 @@
           class: 'feature-container',
         },
       });
+
+      this.infoButton = appendTag(
+        this.featureContainer,
+        createDropdown(this, {
+          id: 'info',
+          lstnrs: {
+            click: () => {
+              this.fetchStatus();
```

Comment:
@rofe It isn't ideal to call fetchStatus since that triggers a bunch of other code, but I do want to make sure that the latest lastModified dates are used.  Any ideas?

Reply:
but isn't fetch status async?

maybe you can hook into the `status-updated` event (I recall that there was evt like that) and make it more _reactive_ :-)

Reply:
@chrischrischris: as @tripodsan points out, `fetchStatus` is async, so you'll have to add a listener for the `statusfetched` event and call `updateModifiedDates` in there. I think it would be ok to call `fetchStatus` on click to force-reload the status (All `statusfetched` listeners must be aware that they could be called multiple times, or else only register once), but bear in mind that the call usually takes 1-2s so there will be a delay in displaying the latest dates. 

I think the sidekick should ideally call `fetchStatus` itself periodically (say, every minute) while open on a page. But that would of course increase the load on the API significantly... @tripodsan any thoughts?

Reply:
> I think the sidekick should ideally call `fetchStatus` itself periodically (say, every minute) while open on a page.

it is not really a problem for the admin-api, but I think the fetch-status that include fetching the last-modified from the sharepoint/gdrive might be a problem due to rate-limiting. so I would not do this.

Reply:
`updateModifiedDates` is called from `enableInfoBtn` which is called in the `statusfetched` listener.

`updateModifiedDates` is also called immediately.  So if the file was recently edited the user may see the old value for 2s then it updates to the new.

We could call `fetchStatus` periodically when the info pane is open, but I don't think it adds much value.  Users can just close/reopen it and it will refresh.